src/adaptors/http2/http2_adaptor.c

```diff
@@ -576,25 +609,27 @@ static int snd_data_callback(nghttp2_session *session,
         memcpy(qd_http2_buffer_cursor(http2_buff), framehd, HTTP2_DATA_FRAME_HEADER_LENGTH);
         qd_http2_buffer_insert(http2_buff, HTTP2_DATA_FRAME_HEADER_LENGTH);
         pn_raw_buffer_t pn_raw_buffs[stream_data->qd_buffers_to_send];
```

Comment:
Ah - a VLA?  Thought you were avoiding these...

Reply:
This VLA is in the fast path. It seems to me that the VLA is the fastest approach in this situation.  Thoughts?

In this case, stream_data->qd_buffers_to_send does not exceed 32.
32 is the number of qd_buffers that will fit in one 16k http2 buffer.